commands/bench/bench.sh

```diff
@@ -26,11 +26,6 @@ output_path="."
 
 profile="production"
 
-if [[ "$repository_name" == "polkadot-sdk" ]]; then
-  output_path="./$target_dir"
```

Comment:
And we dont need this line?

Reply:
oops good catch, it was too easy removal :D